widgetssdk/src/main/java/com/glia/widgets/engagement/EngagementRepositoryImpl.kt

```diff
@@ -206,6 +206,9 @@ internal class EngagementRepositoryImpl(
     }
 
     override fun endEngagement(silently: Boolean) {
+        //This is required to not end the transferred engagement
+        if (isTransferredSecureConversation) return
```

Comment:
I hope this will not make impossible for a visitor to end engagement manually, right?

Reply:
Right!